src/mscorlib/corefx/System/Globalization/CalendarData.Windows.cs

```diff
@@ -139,15 +140,11 @@ internal static int GetCalendars(String localeName, bool useUserOverride, Calend
             }
 
             GCHandle contextHandle = GCHandle.Alloc(data);
-
             try
             {
-                EnumCalendarInfoExExCallback callback = new EnumCalendarInfoExExCallback(EnumCalendarsCallback);
-                Interop.mincore_private.LParamCallbackContext context = new Interop.mincore_private.LParamCallbackContext();
-                context.lParam = (IntPtr)contextHandle;
-
                 // Now call the enumeration API. Work is done by our callback function
-                Interop.mincore_private.EnumCalendarInfoExEx(callback, localeName, ENUM_ALL_CALENDARS, null, CAL_ICALINTVALUE, context);
+                IntPtr callback = AddrofIntrinsics.AddrOf<Func<IntPtr, uint, IntPtr, IntPtr, Interop.BOOL>>(EnumCalendarsCallback);
```

Comment:
We're not using this Windows code in CoreCLR, right?  e.g. does AddrofIntrinsics exist?